app/scripts/directives/inlineEdit.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+angular.module('openshiftConsole').component('inlineEdit', {
+  controller: [
+    InlineEdit
+  ],
+  controllerAs: '$ctrl',
+  bindings: {
+    label: '<',
+    pattern: '<',
+    patternErrmsg: '<',
+    value: '<',
+    saveFunc: '<'
```

Comment:
Does `onEdited` make more sense? Component doesn't know what happens to the value on edit.